service-api/src/main/java/greencity/dto/service/CreateServiceDto.java

```diff
@@ -12,11 +11,19 @@
 @Builder
 @ToString
 public class CreateServiceDto {
-    Integer capacity;
+    @NotNull
+    String name;
+
+    @NotNull
+    String nameEng;
+
+    String description;
```

Comment:
Do we have some validation for `description` and `descriptionEng`?

Reply:
fixed